WS2012R2/lisa/setupscripts/STOR_VHDXResize_Basic.ps1

```diff
@@ -68,6 +70,8 @@ $TC_COVERED = $null
 $TestLogDir = $null
 $TestName   = $null
 $vhdxDrive  = $null
+$controllerType = "SCSI"
```

Comment:
Thank you so much for your comment, I will update it by removing controllerType hardcode here and adding value to xml for all related cases .